source/includes/_appliance_part_type_replacement_linker.md

```diff
@@ -9,6 +9,7 @@ Field | Description
 __appliance_part_type_replacement_linker_id__ <br><font color="DarkGray">_int_</font> <font color="Crimson">__(primary key)__</font> | A unique integer identifier for each appliance_part_type_replacement_linker.
 __<a href="/#appliance-type">appliance_type_id</a>__ <br><font color="DarkGray">_int_</font> <font color="Crimson">(not-null,foreign-key)</font> | 
 __<a href="/#existing-appliance-part-type">existing_appliance_part_type_id</a>__ <br><font color="DarkGray">_int_</font> <font color="Crimson">(not-null,foreign-key)</font> | 
+__max_part_count__  <br><font color="DarkGray">_int_</font> | count of maximum parts that can be changed
```

Comment:
Please add this description: Maximum number of parts of this type that can be replaced for this appliance type

Reply:
Or copy this line of code
`__max_part_count__  <br><font color="DarkGray">_int_</font> <font color="Crimson">(not-null,default=1)</font>| Maximum number of parts of this type that can be replaced for this appliance type`

Reply:
`max_part_count` is nullable field. Please see the response below,
```
{
    "appliance_part_type_replacement_linker_id": 521,
    "appliance_type_id": 27,
    "created_at": "2020-07-20T16:55:23.257503",
    "created_by": "w.theaker@bboxx.co.uk",
    "existing_appliance_part_type_id": 510,
    "max_part_count": null,
    "modified_at": null,
    "modified_by": null,
    "replacement_appliance_part_type_id": 510
}
```

Updating the line to this instead,
`__max_part_count__ <br><font color="DarkGray">_int_</font> <font color="Crimson">(default=1)</font>| Maximum number of parts of this type that can be replaced for this appliance type`